pom.xml

```diff
@@ -859,6 +863,10 @@
         <artifactId>hbase-client</artifactId>
         <version>${hbase10.version}</version>
         <exclusions>
+          <exclusion>
```

Comment:
when I did a dependency analysis, I found only hbase-common and tez-api depends on tools.jar. Do we need to exclude in other dependencies ?

Reply:
Once you exclude it from hbase-common, try to run `dependency:tree` again, you'll see the dependency will get pulled in by some other hbase-* modules.

Reply:
got it, thank you:)